.github/workflows/linux-qe.yml

```diff
@@ -0,0 +1,150 @@
+name: tester-linux
+
+on:
+  workflow_run:
+    workflows: [Build Windows artifacts]
```

Comment:
THis is aligned with previous comment if you create the unified builder flow this should be triggered for the new builder flow